### PR TITLE
Add availability to Ylm holdings

### PR DESCRIPTION
**Pull request by poacherone**
All holdings for sigel Ylm are not for ILL, this script adds availability to all items (that doesn't already have this).

### CODE REVIEW

File: whelktool/scripts/syslib/Ylm-add-availability.groovy [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        holds["hasComponent"].each { it ->
+            if (it["availability"]) {
+                asList(it["availability"]).each { availability ->
+
+                    if (availability["label"] && availability["label"] instanceof String) {
+                        if (!availability["label"].contains("Ej fjärrlån")) {
+                            shouldChange = true
+                        }
+                    }
+                }
```
Comment (olovy) [inline whelktool/scripts/syslib/Ylm-add-availability.groovy:24]: This should be reversed?
Now `shouldChange` is true if there is _any_ availability that is _not_ "Ej fjärrlån"
It should be false if there is _any_ availability that _is_ "Ej fjärrrlån"

**Review by olovy — COMMENTED** [review]
The check for existing availability needs to look for `"@id": "https://id.kb.se/term/enum/NotForILL"? 
Now it only handles local `ItemAvailability`. I'm not sure if there any of these.

**Comment by poacherone** [comment]
There is no linked availability in these records, only a few local ones.

**Review by olovy — APPROVED** [review]
See comment about backwards condition
:ship:

**Comment by poacherone** [comment]
Fixed condition.

**Pull request merged**